plugins.json

```diff
@@ -93,17 +93,23 @@
     "name": "Get Environment Variables",
     "package": "netlify-plugin-get-env-vars",
     "repo": "https://github.com/shortdiv/netlify-plugin-get-env-vars",
-    "version": "1.0.0",
-    "status": "DEACTIVATED"
```

Comment:
The following issue should probably be addressed to make sure this plugin is correctly reporting errors to users: https://github.com/shortdiv/netlify-plugin-get-env-vars/issues/2